### PR TITLE
Re implement ethlance.server.db ns with new tables

### PR DESCRIPTION
**Pull request by jpmonettas**

### CODE REVIEW

File: src/ethlance/server/db.cljs [hunk 1]
```diff
@@ -46,7 +47,329 @@
...
+     [[(sql/call :foreign-key :job/id) (sql/call :references :Job :job/id)]]
+     ]
+    :id-keys [:job/id :job/file-id]
+    :list-keys []}
+
+   {:table-name :Contract
+    :table-columns
+    [[:contract/id :integer]
+     [:job/id :integer]
+     [:contract/status :integer]
```
[the remark this answers is not in the document]
Comment (jpmonettas) [reply src/ethlance/server/db.cljs]: changed all status fields to varchar

**Comment by benzap** [comment]
- Fixes #120

File: src/ethlance/server/db.cljs [hunk 2]
```diff
@@ -46,7 +47,329 @@
...
+     [:category/id :varchar not-nil]
+     ;; PK
+     [(sql/call :primary-key :user/address :category/id)]
+     ;; FKs
+     [[(sql/call :foreign-key :user/address) (sql/call :references :User :user/address)]]
+     [[(sql/call :foreign-key :category/id) (sql/call :references :Category :category/id)]]]
+    :id-keys []
+    :list-keys []}
+
+   {:table-name :ArbiterSkills
```
Comment (benzap) [inline src/ethlance/server/db.cljs]: /s/:ArbiterSkills/:ArbiterSkill

File: src/ethlance/server/db.cljs [hunk 3]
```diff
@@ -46,7 +47,329 @@
...
+    :table-columns
+    [[:user/address address not-nil]
+     [:skill/id :varchar not-nil]
```
Comment (benzap) [inline src/ethlance/server/db.cljs]: :skill/id --> :skill/name

File: src/ethlance/server/db.cljs [hunk 4]
```diff
@@ -46,7 +47,329 @@
...
+     ;; PK
+     [(sql/call :primary-key :user/address :skill/id)]
+     ;; FKs
+     [[(sql/call :foreign-key :user/address) (sql/call :references :User :user/address)]]
+     [[(sql/call :foreign-key :skill/id) (sql/call :references :Skill :skill/id)]]]
```
Comment (benzap) [inline src/ethlance/server/db.cljs]: remove

File: src/ethlance/server/db.cljs [hunk 5]
```diff
@@ -46,7 +47,329 @@
...
+     [:job/id :integer]
+     [:contract/status :integer]
+     [:contract/date-created :unsigned :integer]
+     [:contract/date-updated :unsigned :integer]
+     [:contract/invitation-message-id :integer]
+     [:contract/proposal-message-id :integer]
+     [:contract/proposal-rate :integer]
+     [:contract/proposal-rate-currency-id :varchar]
+     [:contract/raised-dispute-message-id :integer]
+     [:contract/resolved-dispute-message-id :integer]
```
[the remark this answers is not in the document]
Comment (benzap) [reply src/ethlance/server/db.cljs]: From our previous discussion, we are trying to design around the Bounty contracts. We'll have to review this further to see if we can have multiple disputes

**Comment by benzap** [comment]
Lots of little things that need to be fixed, but we can resolve them later. What we have can currently be merged until we discuss it more

**Pull request merged**